fe/fe-core/src/main/java/com/starrocks/common/proc/PartitionsProcDir.java

```diff
@@ -68,38 +71,53 @@ public class PartitionsProcDir implements ProcDirInterface {
     private final PartitionType partitionType;
     private ImmutableList<String> titleNames;
     private Database db;
-    private OlapTable olapTable;
+    private OlapTable table;
     private boolean isTempPartition = false;
 
-    public PartitionsProcDir(Database db, OlapTable olapTable, boolean isTempPartition) {
+    public PartitionsProcDir(Database db, OlapTable table, boolean isTempPartition) {
         this.db = db;
-        this.olapTable = olapTable;
+        this.table = table;
         this.isTempPartition = isTempPartition;
-        this.partitionType = olapTable.getPartitionInfo().getType();
+        this.partitionType = table.getPartitionInfo().getType();
         this.createTitleNames();
     }
 
     private void createTitleNames() {
-        ImmutableList.Builder<String> builder = new ImmutableList.Builder<String>()
-                .add("PartitionId").add("PartitionName")
-                .add("VisibleVersion").add("VisibleVersionTime").add("VisibleVersionHash")
-                .add("State").add("PartitionKey");
-
-        if (this.partitionType == PartitionType.LIST) {
-            builder.add("List");
-        } else {
-            builder.add("Range");
-        }
-
-        builder.add("DistributionKey").add("Buckets").add("ReplicationNum");
-        if (olapTable.isLakeTable()) {
-            builder.add("EnableStorageCache").add("StorageCacheTtlSecond").add("AllowAsyncWriteBack")
```

Comment:
why remove cache and async write back properties?

Reply:
fixed, PTAL